Code/Editor/EditorToolsApplication.cpp

```diff
@@ -105,7 +105,17 @@ namespace EditorInternal
         AzFramework::Application::StartupParameters params;
 
         // Must be done before creating QApplication, otherwise asserts when we alloc
-        AzToolsFramework::ToolsApplication::Start({}, params);
+        AzToolsFramework::ToolsApplication::Start({
+                /* .m_useExistingAllocator = */ false,
+                /* .m_allocationRecordsSaveNames = */ false,
+                /* .m_allocationRecordsAttemptDecodeImmediately = */ false,
+                /* .m_autoIntegrityCheck = */ false,
+                /* .m_markUnallocatedMemory = */ false,
+                /* .m_doNotUsePools = */ false,
+                /* .m_enableScriptReflection = */ false,
```

Comment:
Should "enableScriptReflection" be enabled?

Reply:
Fixed.